### PR TITLE
Feat/qr generator

### PR DESCRIPTION
**Pull request by bennchong**
QR Generator indicates activity when uploading
Links up with transferAPI to upload workpass

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "expo-file-system": "^5.0.1",
     "expo-permissions": "^5.0.1",
     "expo-secure-store": "^5.0.1",
+    "isomorphic-fetch": "^2.2.1",
```
Comment (yehjxraymond) [inline package.json]: Use Axios. Remove pls

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=25

File: src/services/transferAPI.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import Axios from "axios";
```
Comment (samuellye98) [inline src/services/transferAPI.tsx]: Why is this capital A!!

File: package.json (hunk 1 above)
Comment (bennchong) [reply package.json]: Removed

File: src/services/transferAPI.tsx (hunk 2 above)
Comment (bennchong) [reply src/services/transferAPI.tsx]: Changed

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=25

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=25

**Pull request merged**